pom.xml

```diff
@@ -41,6 +41,21 @@
                  <goals>deploy</goals>
              </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.codehaus.gmaven</groupId>
+            <artifactId>gmaven-plugin</artifactId>
+                <version>1.5-jenkins-3</version>
+            <configuration>
+                <providerSelection>1.7</providerSelection>
+            </configuration>
+            <dependencies>
+                <dependency>
+                    <groupId>org.codehaus.gmaven.runtime</groupId>
+                    <artifactId>gmaven-runtime-1.7</artifactId>
+                    <version>1.3</version>
+                </dependency>
+            </dependencies>
+          </plugin>
```

Comment:
The default dependency is 1.5-jenkins-1. Groovy tests cannot be compiled with it. Short googling indicates a conflict with groovy-all.
http://stackoverflow.com/questions/9223689/solve-error-java-lang-nosuchmethoderror-org-codehaus-groovy-ast-modulenode-gets